src/devices/MotorHat/README.md

```diff
@@ -0,0 +1,21 @@
+﻿# MotorHat
+
+## Summary
+The MotorHat is a Raspberry Pi Motor Hat based on PCA9685 PWM controller, that uses I²C-bus.
+
+Allows controlling:
+* Up to 4 DC Motors or Up to 2 Stepper Motors
+* Up to 4 PWM Channels, (can be used to control any PWM device, including ServoMotors)
+
+## Device Family
+
+**Adafruit Motor Hat**: https://www.adafruit.com/product/2348
```

Comment:
I'd remove this section and add a link somewhere in the summary instead, like so: `[Adafruit Motor Hat](https://www.adafruit.com/product/2348)`